diagrams/metametamodel-gen.puml

```diff
@@ -50,23 +45,23 @@ abstract class NamespacedEntity {
   qualifiedName(): String
 }
 
-class PrimitiveType extends Datatype
+class PrimitiveType extends DataType
 
 class Property extends Feature
 
 class Reference extends Link
 
 ' primitive type: "String"
 
-class Typedef extends Datatype
+class Typedef extends DataType
```

Comment:
I think we agreed on having Typedefs, however I cannot see them in the last version of the diagram I can see here https://github.com/LIonWeb-org/organization/blob/main/lioncore/metametamodel.svg

We should figure out which ways to go and make it consistent

Reply:
I'll check with that location.

Reply:
I've removed `Typedef`.